##### TAG DEFINITION OF ENVIRONMENT #####
# [ID, SIZE(m), X(m), Y(m), Z(m), psi(Deg)]
TAG_TRUE_RT:
  TAGS:
    - [256, 0.16, 0.50, 0.00, 0.800, 0.0]
    - [257, 0.16, -0.1, 0.75, 1.000, 95.0]
    - [258, 0.16, 0.25, 0.75, 0.900, 75.0]